.NET-example/README.md

```diff
@@ -2,19 +2,19 @@
 
 ## Auto-Instrumentation
 
-We will go through a working example of a .net application auto-instrumented with OpenTelemetry. To keep things simple,
+We will go through a working example of a .NET application auto-instrumented with OpenTelemetry. To keep things simple,
 we will create a basic “Hello World” web application, instrument it with OpenTelemetry library to generate trace data
 and send it to an OpenTelemetry Collector. The Collector will then export the trace data to the Wavefront Proxy which
 will eventually export the trace data to the Tanzu Observability UI.
 
 ![Here is how it works:](https://raw.githubusercontent.com/wavefrontHQ/opentelemetry-examples/main/TraceFlow.png?raw=true)
 
-If you have not set up an OpenTelemetry Collector or Wavefront proxy yet, then check
+If we have not set up an OpenTelemetry Collector or Wavefront proxy yet, then check
 out [this guide](https://github.com/wavefrontHQ/opentelemetry-examples/blob/main/README.md).
 
-#### Step 1: Get your example application
+#### Step 1: Get our example application
 
-You can easily instrument your application, but if you do not have one then refer to a following simple application. Our
+We can easily instrument our any other application, but if we do not have one then refer to a following simple application. Our
```

Comment:
This sentence reads a little oddly now. Maybe something like 
> The instrumentation works with any application - for this walkthrough we'll refer to the following simple application.

would read better?

Reply:
I have updated the code. Please review once again.